all_protos/exchange/injective_chart_rpc.proto

```
@@ -0,0 +1,96 @@
+// Code generated with goa v3.7.0, DO NOT EDIT.
+//
+// InjectiveChartRPC protocol buffer definition
+//
+// Command:
+// $ goa gen github.com/InjectiveLabs/injective-indexer/api/design -o ../
+
+syntax = "proto3";
+
+package injective_chart_rpc;
```

Buf (1.47.2)

Files with package "injective_chart_rpc" must be within a directory "injective_chart_rpc" relative to root but were in directory "all_protos/exchange".

(PACKAGE_DIRECTORY_MATCH)